litellm/llms/vllm/completion/handler.py

```diff
@@ -27,14 +27,31 @@ def __init__(self, status_code, message):
 
 
 # check if vllm is installed
-def validate_environment(model: str):
+def validate_environment(model: str, optional_params: Union[dict, None]):
```

Comment:
this should be implemented in the transformation.py - https://github.com/BerriAI/litellm/blob/246e3bafc89e19297fa42aabbfca6d25058e5989/litellm/llms/vllm/completion/transformation.py#L14